python/docs/source/user_guide/pandas_on_spark/pandas_pyspark.rst

```diff
@@ -114,8 +114,8 @@ Spark DataFrame can be a pandas-on-Spark DataFrame easily as below:
    2   8
    3   9
 
-However, note that it requires to create new default index in case pandas-on-Spark DataFrame is created from
-Spark DataFrame. See `Default Index Type <options.rst#default-index-type>`_. In order to avoid this overhead, specify the column
+However, note that a new default index is created when a pandas-on-Spark DataFrame is created from
```

Comment:
I wouldn't try to fix "a pandas-on-Spark DataFrame" and "a Spark DataFrame" here alone .. there are too many instances like this to fix in the whole documentation.

Reply:
Note and changed. Thanks for your help @HyukjinKwon!